action.yaml

```diff
@@ -34,6 +34,18 @@ inputs:
     default: 41
     required: false
     type: integer
+  github_token:
+    description: |
+      The github access token (secrets.GITHUB_TOKEN) to create pull request on Github.
+      Required if running in migration mode.
+    default: ${{ github.token }}
+    required: false
```

Comment:
Tested the default parameters, they don't work (I thought it would, I've referred it from create-pull-request actions). Removed for clarity.
Thank you for the feedback!